docs/openapi.yaml

```diff
@@ -488,6 +488,49 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Problem"
+
+  /import-candidate/{identifier}:
+    get:
+      summary: Fetch Import candidate by Identifier
+      tags:
+        - external
+      parameters:
+        - in: path
+          name: identifier
+          schema:
+            type: string
+            format: uuid
+          required: true
+          description: UUID identifier of the import candidate to fetch.
+      x-amazon-apigateway-integration:
+        uri:
+          Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${NvaFetchImportCandidateFunction.Arn}:live/invocations
+        responses: { }
+        httpMethod: POST
+        type: AWS_PROXY
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Publication'
+            application/ld+json:
+              schema:
+                $ref: '#/components/schemas/Publication'
+            application/vnd.datacite.datacite+xml:
+              schema:
+                $ref: '#/components/schemas/PublicationDataCiteXml'
+            application/vnd.schemaorg.ld+json:
+              schema:
+                $ref: '#/components/schemas/PublicationSchemaOrg'
+        '410':
+          description: Permanently gone
```

Comment:
I don't think we will logically delete stuff, only physically delete candidates.

Resource service uses this response if a publication has status = deleted.

An import candidate doesn't have that status field.